manifests/init.pp

```diff
@@ -222,6 +559,150 @@
     }
   }
 
+  if type($enable_db_audit_user) == 'boolean' {
```

Comment:
You don't need a separate PR to make changes to this one. Just push your changes to this branch and they will show up here.